src/main/scala/org/apache/sysml/api/dl/Utils.scala

```diff
@@ -111,6 +111,30 @@ object Utils {
 	  }
 	}
 	
+	class CopyCaffeDeconvFloatToSystemMLDeconvDoubleArray(data:java.util.List[java.lang.Float], F:Int, C:Int, H:Int, W:Int, arr:Array[Double]) 
+	    extends CopyFloatToDoubleArray(data, C, F*H*W, false, arr) {
+	  override def run(): Unit = {
```

Comment:
Can you add a quick note that this is converting from shape `(F, C, H, W)` to `(C, F, H, W)`?